client/my-sites/plans/jetpack-plans/product-store/featured-item-card/index.tsx

```diff
@@ -49,9 +49,9 @@ export const FeaturedItemCard: React.FC< FeaturedItemCardProps > = ( {
 									href="#"
 									plain
 								>
-									{ translate( 'More about %(product)s', {
-										args: {
-											product: title,
+									{ translate( 'More about {{productName/}}', {
+										components: {
+											productName: <>{ title }</>,
```

Comment:
It's possible that `title` can be React element. In that case the link text does not show up correctly.